### ISSUE TITLE
Chrome - GCcollab login appears in tab order

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/TalentCloud#565]
# Description
When tabbing through our homepage, after the "our team" section, the focus goes to some elements that are not visible. These appear to be from the old GCcollab login. The elements appear to have aria-hidden=true, but are still receiving focus. Suggest to remove this markup completely.
